redash/query_runner/dgraph.py

```diff
@@ -0,0 +1,160 @@
+import json
+
+try:
+    import pydgraph
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.utils import json_dumps, json_loads
+
+
+def to_string(s):
+    """From https://github.com/vinay20045/json-to-csv"""
+    try:
+        return str(s)
+    except:
+        # Change the encoding type if needed
+        return s.encode('utf-8')
+
+
+def reduce_item(reduced_item, key, value):
+    """From https://github.com/vinay20045/json-to-csv"""
+    # Reduction Condition 1
+    if type(value) is list:
+        i = 0
+        for sub_item in value:
+            reduce_item(reduced_item, key+'/'+to_string(i), sub_item)
+            i = i+1
+
+    # Reduction Condition 2
+    elif type(value) is dict:
+        sub_keys = value.keys()
+        for sub_key in sub_keys:
+            reduce_item(reduced_item, key+'/'+to_string(sub_key), value[sub_key])
+
+    # Base Condition
+    else:
+        reduced_item[to_string(key)] = to_string(value)
```

Comment:
I would just go with `reduced_item[key] = value` unless `key` might be non string, then I would consider casting it. 

Why convert all the values to strings?

Reply:
probably just a copy/paste thing.  will test and verify it still works